src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -578,30 +587,28 @@
 			};
 
 			/**
-			 * Return offset of element
-			 * @method getOffsetOfElement
+			 * Return top offset of element
+			 * @method getTopOffsetOfElement
```

Comment:
You modified description but method name remains old.

Reply:
Done.